src/Alpha_complex/test/Alpha_complex_dim3_unit_test.cpp

```diff
@@ -0,0 +1,117 @@
+/*    This file is part of the Gudhi Library - https://gudhi.inria.fr/ - which is released under MIT.
+ *    See file LICENSE or go to https://gudhi.inria.fr/licensing/ for full license details.
+ *    Author(s):       Vincent Rouvreau
+ *
+ *    Copyright (C) 2015 Inria
+ *
+ *    Modification(s):
+ *      - YYYY/MM Author: Description of the modification
+ */
+
+#define BOOST_TEST_DYN_LINK
+#define BOOST_TEST_MODULE "alpha_complex"
```

Comment:
It doesn't seem to bother the execution, nor the test, but I think we should set unique name for `BOOST_TEST_MODULE`.

```suggestion
#define BOOST_TEST_MODULE "alpha_complex_dim3"
```